x/auth/client/cli/tx_multisign.go

```diff
@@ -29,6 +29,80 @@ type BroadcastReq struct {
 	Mode string         `json:"mode" yaml:"mode"`
 }
 
+// GetMultiMsgSignCommand returns the multi-msg-sign cmd
+func GetMultiMsgSignCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "multi-msg-sign [file] [file]...",
+		Aliases: []string{"multimsgsign"},
+		Short:   "Combine messages from transactions which are generated offline.",
+		PreRun:  preSignCmd,
+		Example: fmt.Sprintf("%s tx multi-msg-sign tx1.json tx2.json tx3.json ", version.AppName),
+		Long: strings.TrimSpace(
+			fmt.Sprintf(
+				`Combine the transactions created with the --generate-only flag and Sign those transactions.
+
+Example: 
+$ %s tx multi-msg-sign tx1.json tx2.json tx3.json 
+
+It will read a transactions from [files], sign it, and print its JSON encoding.
+3EFBCD425B719FD8CF960552D33E7721C26B0114D33E49D5EBD79E3B70618775
```

Comment:
why is this hash here? Typo?

Reply:
May be he is printing a sample json econding for the tx, which is not required. @gsk967 you can just re-word it as `tx hash` instaed of `its JSON econding.` above as that would be more understandable for users.